kernel/comps/virtio/src/transport/mod.rs

```diff
@@ -24,16 +24,17 @@ pub mod pci;
 pub trait VirtioTransport: Sync + Send + Debug {
     // ====================Device related APIs=======================
 
-    fn device_type(&self) -> VirtioDeviceType;
+    /// Get device type.
+    fn get_device_type(&self) -> VirtioDeviceType;
 
     /// Get device features.
-    fn device_features(&self) -> u64;
+    fn read_device_features(&self) -> u64;
 
     /// Set driver features.
     fn set_driver_features(&mut self, features: u64) -> Result<(), VirtioTransportError>;
 
     /// Get device status.
-    fn device_status(&self) -> DeviceStatus;
+    fn read_device_status(&self) -> DeviceStatus;
 
     /// Set device status.
     fn set_device_status(&mut self, status: DeviceStatus) -> Result<(), VirtioTransportError>;
```

Comment:
Another important consideration in naming is consistency. The verbs _get_ should be paired with _set_, _load_ with _store_,  and _read_ with _write_. So this setter method should be renamed to `write_device_status`.